<ui version="4.0" >
 <class>YaPreferences</class>
 <widget class="QWidget" name="YaPreferences" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="leftMargin" >
    <number>10</number>
   </property>
   <property name="topMargin" >
    <number>10</number>
   </property>
   <property name="rightMargin" >
    <number>10</number>
   </property>
   <property name="bottomMargin" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>7</number>
     </property>
     <item>
      <widget class="YaPreferencesTabButton" name="preferencesButton" >
       <property name="text" >
        <string>Preferences</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="YaPreferencesTabButton" name="manageAccountsButton" >
       <property name="text" >
        <string>Manage Accounts</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="preferencesPage" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>696</width>
        <height>488</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="preferencesFrame" >
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" >
          <property name="topMargin" >
           <number>7</number>
          </property>
          <property name="spacing" >
           <number>0</number>
          </property>
          <item row="0" column="0" >
           <layout class="QVBoxLayout" name="verticalLayout_2" >
            <property name="spacing" >
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="groupLabelContactList" >
              <property name="text" >
               <string>Contact List</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="useOfficeBackground" >
              <property name="text" >
               <string>Use "office" window background</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ctrlEnterSendsChatMessages" >
              <property name="text" >
               <string>Ctrl+Enter shortcut sends chat messages</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="showContactListGroups" >
              <property name="text" >
               <string>Show groups in contact list</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="spacing" >
               <number>10</number>
              </property>
              <property name="leftMargin" >
               <number>0</number>
              </property>
              <item>
               <widget class="QCheckBox" name="enableContactListAvatars" >
                <property name="text" >
                 <string>Avatars:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="contactListAvatarsComboBox" >
                <property name="maximumSize" >
                 <size>
                  <width>16777215</width>
                  <height>16</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>0</width>
                  <height>10</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="2" >
           <layout class="QVBoxLayout" >
            <property name="spacing" >
             <number>5</number>
            </property>
            <item>
             <widget class="QCheckBox" name="startAutomatically" >
              <property name="text" >
               <string>Start automatically when starting Windows</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="groupLabelNotifications" >
              <property name="text" >
               <string>Notifications</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="showMoodChangePopups" >
              <property name="text" >
               <string>Show mood change popups</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="showMessageNotifications" >
              <property name="text" >
               <string>Show incoming message popups</string>
              </property>
             </widget>
            </item>
            <item>
              <layout class="QHBoxLayout" >
               <property name="leftMargin" >
                <number>18</number>
               </property>
               <item>
             <widget class="QCheckBox" name="showMessageText" >
              <property name="text" >
               <string>Show message text</string>
              </property>
             </widget>
               </item></layout>
            </item>
            <item>
             <widget class="QCheckBox" name="playSounds" >
              <property name="text" >
               <string>Play sounds</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer" >
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item rowspan="4" row="0" column="3" >
           <widget class="QFrame" name="expandingFillerFrame_2" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>22</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0" >
           <layout class="QVBoxLayout" name="verticalLayout" >
            <property name="spacing" >
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="groupLabelChat" >
              <property name="text" >
               <string>Chat window</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" >
              <property name="spacing" >
               <number>10</number>
              </property>
              <property name="leftMargin" >
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="labelChatBackground" >
                <property name="text" >
                 <string>Background:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="chatBackgroundComboBox" >
                <property name="maximumSize" >
                 <size>
                  <width>16777215</width>
                  <height>16</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>0</width>
                  <height>10</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="enableEmoticons" >
              <property name="text" >
               <string>Enable emoticons</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" >
              <property name="spacing" >
               <number>10</number>
              </property>
              <item>
               <widget class="QCheckBox" name="logMessageHistory" >
                <property name="text" >
                 <string>Log message history</string>
                </property>
                <property name="tristate" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="YaLinkButton" name="clearMessageHistory" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>10000</width>
                  <height>15</height>
                 </size>
                </property>
                <property name="text" >
                 <string>Clear Message History</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>0</width>
                  <height>10</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" >
              <property name="horizontalSpacing" >
               <number>3</number>
              </property>
              <property name="verticalSpacing" >
               <number>2</number>
              </property>
              <item row="0" column="0" colspan="3" >
               <widget class="QLabel" name="fontLabel" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>Use the following font in chat window:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" >
               <widget class="QFontComboBox" name="fontCombo" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize" >
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>178</width>
                  <height>16</height>
                 </size>
                </property>
                <property name="editable" >
                 <bool>false</bool>
                </property>
                <property name="frame" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="1" >
               <widget class="QComboBox" name="fontSizeCombo" >
                <property name="maximumSize" >
                 <size>
                  <width>16777215</width>
                  <height>16</height>
                 </size>
                </property>
                <property name="editable" >
                 <bool>false</bool>
                </property>
                <item>
                 <property name="text" >
                  <string>6</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>8</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>9</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>10</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>11</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>12</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>14</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>16</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="2" >
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>0</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="2" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>21</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>191</width>
              <height>122</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="6" column="0" colspan="4" >
           <widget class="QFrame" name="expandingFillerFrame" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="topMargin" >
          <number>8</number>
         </property>
         <item>
          <widget class="YaPushButton" name="yaOnlinePreferences" >
           <property name="text" >
            <string>Ya.Online Preferences</string>
           </property>
           <property name="icon" >
            <iconset resource="yastuff.qrc" >
             <normaloff>:/images/user-actions/button_glyph_online.png</normaloff>:/images/user-actions/button_glyph_online.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>111</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="YaPushButton" name="okButton" >
           <property name="text" >
            <string>Close</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>YaPushButton</class>
   <extends>QPushButton</extends>
   <header>yapushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>YaLinkButton</class>
   <extends>QPushButton</extends>
   <header>yalinkbutton.h</header>
  </customwidget>
  <customwidget>
   <class>YaPreferencesTabButton</class>
   <extends>QPushButton</extends>
   <header>yapreferencestabbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="yastuff.qrc" />
 </resources>
 <connections/>
</ui>
